spec/requests/posts/index_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :system do
+  describe 'GET /users/:id/posts' do
+    it 'should show the posts index page' do
+      get '/users/11/posts'
+      expect(response).to be_successful
+    end
+
+    it 'should render correct template' do
+      get '/users/11/posts'
+      expect(response.body).to include('index page for posts')
+    end
+
+    it 'should return correct data' do
+      get '/users/11/posts'
+      expect(response.body).to have_selector('h1', text: 'This is the index page for posts')
+    end
```

Comment:
- [ ] In this test suite we are checking the response body includes the correct placeholder text, currently you have a have_selector matcher which is available due to type: :system it is a capybara thing, once you change the type to request it will throw an error since it is not available in controller testing, so a more appropriate check would be like :point_down: 

`expect(response.body).to include('This is the index page for posts')`

- [ ] Please do the same for `spec/requests/posts/show_spec.rb`, `spec/requests/users/index_spec.rb`, `spec/requests/users/show_spec.rb`